apps/ops/models/job.py

```diff
@@ -181,13 +181,9 @@ def date_last_run(self):
     @property
     def summary(self):
         summary = {
-            "total": 0,
-            "success": 0,
+            "total": self.executions.count(),
+            "success": self.executions.filter(status=JobStatus.success).count(),
         }
-        for execution in self.executions.all():
-            summary["total"] += 1
-            if execution.is_success:
-                summary["success"] += 1
         return summary
 
     @property
```

Comment:
Since it is an existing piece of code that has already been tested, I am not aware of any specific issues. However, if there are no known problems with the code currently, we can make some general optimizations such as reducing unnecessary calculations and removing unused properties to improve its efficiency:

1. Remove `@property` from `summary`, since it does not seem to change during runtime but rather updates in response to certain events.

2. If necessary, optimize or refactor functions for readability, performance improvements, and potentially more advanced logic (e.g., caching or using pagination instead).

The current version appears suitable according to what you've described, so no further actions would be appropriate unless new requirements arise from changing usage scenarios, API changes, security concerns, external libraries being updated... etc!